mne/viz/_scraper.py

```diff
@@ -52,15 +52,14 @@ def __call__(self, block, block_vars, gallery_conf):
 
 @contextmanager
 def _screenshot_mode(browser):
-    if hasattr(browser.mne, "toolbar"):  # if there is an error, it might not
-        browser.mne.toolbar.setVisible(False)
-    browser.statusBar().setVisible(False)
+    need_zen = browser.mne.scrollbars_visible
+    if need_zen:
```

Comment:
not a fan of the walrus operator?
```suggestion
    if need_zen := browser.mne.scrollbars_visible:
```

Reply:
(somewhat tongue-in-cheek suggestion, feel free to ignore)